examples/tutorials/Part 11 bis - Encrypted inference on ResNet-18.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Q: Maybe it is worth to show how encrypted_data and encrypted_model look?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I didn't want to explain in depth fixed precision and secret sharing for people who just want to "encrypt" (and don't want to read section 1), so I didn't want to both by printing the stuff</p>